pandas/io/gbq.py

```diff
@@ -407,6 +465,10 @@ def to_gbq(dataframe, destination_table, project_id=None, chunksize=10000,
     connector = GbqConnector(project_id, reauth = reauth)
     dataset_id, table_id = destination_table.rsplit('.',1)
 
+    #if table does not exist, create new table
```

Comment:
- what happens if an existing table has a different schema?
- sql has this keyword ``if_exists``

```
if_exists : {'fail', 'replace', 'append'}, default 'fail'
    - fail: If table exists, do nothing.
    - replace: If table exists, drop it, recreate it, and insert data.
    - append: If table exists, insert data. Create if does not exist.
```

 I think you need it here (can you support ``replace``?)